src/utils/helpers.js

```diff
@@ -0,0 +1,9 @@
+export const filteredMovieData = movies => {
```

Comment:
Thank you for setting up this utils and helper folder, this will be good practice utilizing these.